executor/cook/executor.py

```diff
@@ -252,6 +254,40 @@ def get_task_state(exit_code):
         return cook.TASK_FINISHED
 
 
+def set_environment(environment, key, value):
+    """Updates an entry in the environment dictionary.
+
+    Returns
+    -------
+    Nothing.
+    """
+    if key not in environment or environment[key] != value:
+        logging.info('Setting process environment[{}]={}'.format(key, value))
+        environment[key] = value
+
+
+def retrieve_process_environment(config, os_environ):
+    """Prepares the environment for the subprocess.
+    The function also ensures that env[EXECUTOR_PROGRESS_OUTPUT_FILE_ENV] is set to config.progress_output_name.
```

Comment:
Should this comment refer to `config.progress_output_env_variable` instead of `EXECUTOR_PROGRESS_OUTPUT_FILE_ENV`?

Reply:
Done.